src/clients/subgraph/queries/getAssetsInAccount/assetsInAccountQuery.ts

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'react-query';
+
+import FunctionKey from 'constants/functionKey';
+
+import subgraphClient from '../../subgraphClient';
+
+const useGetAssetsInAccountSubgraph = (accountAddress: string) =>
+  useQuery([FunctionKey.GET_ASSETS_IN_ACCOUNT, { accountAddress }], () =>
+    subgraphClient.AssetsInAccountQuery({ accountAddress }),
+  );
+
+export default useGetAssetsInAccountSubgraph;
```

Comment:
I'd rename this file to `useGetAssetsInAccountSubgraph` since it only exports the hook of the same name.